### DISCUSSION TITLE
Label each method for client or server

### DISCUSSION THREAD

**Discussion by hawarhekmat** [supabase/supabase#36081] [Feature Requests]
in the documentations there is no way to tell the method is for server or client or both, so sometimes it will be hard to write code, like in the nextjs server actions. Sometimes when writing down the client only code in the server actions it works but it doesn't do anything and sometimes it won't work so finding the problem it will be hard without knowing if the code only supports client or server.

**Comment by Hallidayo** [comment]
Hi @hawarhekmat - Have you got some example pages where to add these labels?

**Comment by hawarhekmat** [comment]
> Hi @hawarhekmat - Have you got some example pages where to add these labels?

https://supabase.com/docs/reference/javascript/select

next to fetch data add if this method is only (client or server) and so on for each of them
and if it supports server actions or not

**Comment by Anurag15v** [comment]
@hawarhekmat Hi 
Can you provide a bit more clarifications. Just required to check if select, insert ... works on both client and server actions of nextjs ?

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions/enhancements.
